content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
+    public IEnumerable<Todo> GetTodoList()
+    {
+        return new Todo[]
+        {
+            new Todo{
```

Comment:
Very minor, not sure this is from a code formatting in your IDE but please add a space before open curry brace.

Reply:
Why?

Reply:
I think when it time to create a property initialize, VS studio or VS Code usually format code like.
```
var todo = new Todo 
{

};
```

I found myself having a space before `{` is easier to read for me.
I think I am wrong for this because if we consistency with method, e.g
```
void MyMethod(){
}
then  we should apply the same format for a property initializeation.
var todo = new Todo{
};
```
So please keep this one. Sorry for my mistake.

Reply:
Ok, ty